modules/redpanda/mounts/redpanda.yaml.tpl

```diff
@@ -10,6 +10,13 @@ redpanda:
       port: 9092
       authentication_method: {{ .KafkaAPI.AuthenticationMethod }}
 
+    # This listener is required to enable connection from within the Docker
+    # network
+    - address: 0.0.0.0
+      name: docker
+      port: 29092
+      authentication_method: {{ .KafkaAPI.AuthenticationMethod }}
```

Comment:
In java, [adding multiple listeners is allowed](https://github.com/testcontainers/testcontainers-java/blob/main/modules/redpanda/src/main/resources/testcontainers/redpanda.yaml.ftl#L24-L29) because it can be shared through the docker network and via a proxy such as toxiproxy. I think the same approach can be offered in go. For the test, look at this [one](https://github.com/testcontainers/testcontainers-java/blob/main/modules/redpanda/src/test/java/org/testcontainers/redpanda/RedpandaContainerTest.java#L114-L142) using kcat container.

Reply:
@eddumelendez I have update the PR to be more aligned with the implementation you linked, mind to check it out ?, thanks